src/@next/Combobox/components/OptionList/OptionListStyle.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { borderRadius4 } from '../../../utilities/borderRadius';
+import { Blue, Neutral } from '../../../utilities/colors';
+import { space4, space8 } from '../../../utilities/spacing';
+
+interface OptionListContainerProps {
+  textInputWidth?: number;
+}
+
+export const OptionListContainer = styled.div<OptionListContainerProps>`
```

Comment:
the position of OptionListContainer doesn't align with TextInput when the screen width is smaller than the component's width. Could you take a look at this issue?

Reply:
@chiahou it had to do with how we defined the width for TextInput. thanks for noticing. I've addressed above. You'll need to refresh when you try to change your screen size to make sure both have same width